lib/web_ui/dev/test_runner.dart

```diff
@@ -371,6 +380,17 @@ class TestCommand extends Command<bool> with ArgUtils {
       isFirefoxIntegrationTestAvailable ||
       isSafariIntegrationTestAvailable;
 
+  // Whether the tests will do screenshot testing.
+  bool get isScreenhotTestsAvailable =>
+      isIntegrationTestsAvailable || isUnitTestsScreenshotsAvailable;
+
+  // For unit tests screenshot tests and smoke tests only run on:
+  // "Chrome/iOS" for LUCI/local.
+  bool get isUnitTestsScreenshotsAvailable =>
+      ((isChrome && isLuci && io.Platform.isLinux) ||
+          (isChrome || isSafariIOS) && !isLuci) ||
+      (isSafariIOS && isLuci);
```

Comment:
I think it's the same as:

```
  isChrome && (io.Platform.isLinux || !isLuci) || isSafariIOS
```

IOW, for Chrome-Linux we always run screenshots. On Windows/macOS we run Chrome screenshots locally. We also always run screenshots on iOS Safari.

Reply:
[Wolfram Alpha](https://www.wolframalpha.com/input/?i=%28%28a+%26%26+b+%26%26+c%29+%7C%7C+%28a+%7C%7C+d%29+%26%26+%21b%29+%7C%7C+d+%26%26+b) agrees :P

`isChrome && !isLuci || isChrome && io.Platform.isLinux || isSafariIOS `

Reply:
Ah thanks for the comment :smile: (I loved the link :heart: )

I believe my version is easier to edit, since the lines that contains `isLuci` are separate. Therefore one can remove them on/off easily. Having said that I'm probably the only person in the team that runs PRs on `led` so it is more important that this is succinct.